integration-cli/fixtures_linux_daemon_test.go

```diff
@@ -46,7 +44,7 @@ func ensureSyscallTest(c *check.C) {
 	if !doIt {
 		return
 	}
-	protectedImages["syscall-test:latest"] = struct{}{}
+	defer testEnv.ProtectImage(c, "syscall-test:latest")
```

Comment:
@dnephin oh you are right. It was required to be defered in the previous PR because I was inspecting the image (and thus needed to wait for the image to be actually imported) — it's not the case anymore, yet 👼 .